src/anm/media/audio.js

```diff
@@ -47,7 +47,7 @@ var audioContext = getAudioContext();
  * @class anm.Audio
  */
 function Audio(url) {
-    this.url = url + audioExt;
+    this.url = /\.\w+$/i.test(url) ? url : url + audioExt;
```

Comment:
@skavish As I recall, there was a high chance that two sources (both `ogg` and `mp3`) exist on server and that's why we used audio URL w/o extension — this way we have chosen what to play by checking if user's browser supports this format, not by dictating it to user.